README.md

```diff
@@ -1 +1,62 @@
-# Awesome-books-with-ES6
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# awesome-books-project-modules
+
+
+![](file:///home/moise/my-awesome-books-modules/index.html)
+
+# Awesome books: plain JavaScript with objects
+
+> This my awesome books project using modules to separate JavaScript files into small chunks. 
+
+![screenshot](awesome-book-img/awesome-book.png).
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript ES6/Modules
+- Visual Studio Code.
+- Linters
+
+## Live Demo
+
+[github page](https://moise-mulungu.github.io/Awesome-books-with-ES6/)
+
+[live demo](http://127.0.0.1:5500/index.html)
+
+## Getting Started
+
+To get a local copy of this app you can use :
+```
+git clone  https://github.com/moise-mulungu/Awesome-books-with-ES6/tree/create-es6-branch
```

Comment:
- [ ] Let's try to be detailed in our documentation as it can be of great benefit to users. This link is not correct for cloning the repo. This is the correct one: `https://github.com/moise-mulungu/Awesome-books-with-ES6.git`. Kindly fix this.